src/intl/fr.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Test file to ensure translation support works, this shouldn't actually be committed.